tekton/utils_test.go

```diff
@@ -84,7 +88,7 @@ var _ = Describe("Utils", func() {
 	Context("when using utility functions on PipelineRun objects", func() {
 		It("is a PipelineRun object and contains the required labels that identifies it as one", func() {
 			Expect(isReleasePipelineRun(releasePipelineRun.
-				WithReleaseLabels(release.Name, release.Namespace).
+				WithReleaseLabels(release.Name, release.Namespace, release.GetAnnotations()[logicalcluster.AnnotationKey]).
```

Comment:
Is this set automatically when you run on kcp?

Reply:
yes, it seems to be